tests/kolainst/nondestructive/misc.sh

```diff
@@ -66,6 +66,20 @@ echo "ok status text"
 runuser -u bin rpm-ostree status
 echo "ok status doesn't require active PAM session"
 
+# Verify we work without polkit, as root
+systemctl mask --now polkit
+systemctl restart rpm-ostreed
+# This should work as root
+rpm-ostree reload
+# And non-root should still work for methods that don't need auth
+runuser -u bin rpm-ostree status
+# But this shouldn't work
+if runuser -u core rpm-ostree reload; then
+    assert_not_reached "Was able to reload as non-root!"
+fi
```

Comment:
Probably worth checking the error is what we expect here so it's testing what we think it's testing.

Reply:
Also worth trying with `rpm-ostree install` too to exercise the OS interface side.